### ISSUE TITLE
Can't install jupyter-lsp-python-lsp-server and jupyter-lsp-python-plugins

### ISSUE THREAD

**Issue by maresb** [conda-forge/jupyter-lsp-feedstock#25]
It looks like `pyls-black` [requires `python-language-server`](https://github.com/conda-forge/pyls-black-feedstock/blob/64b2b3db8438a8a437516382a36d9176422ab839/recipe/meta.yaml#L23) so then you wind up with the nasty pinning problems with `parso` etc.

```bash
$ docker run --rm -it condaforge/mambaforge
(base) root@6620e96293ad:/# mamba install jupyter-lsp-python-lsp-server jupyter-lsp-python-plugins

                  __    __    __    __
                 /  \  /  \  /  \  /  \
                /    \/    \/    \/    \
███████████████/  /██/  /██/  /██/  /████████████████████████
              /  / \   / \   / \   / \  \____
             /  /   \_/   \_/   \_/   \    o \__,
            / _/                       \_____/  `
            |/
        ███╗   ███╗ █████╗ ███╗   ███╗██████╗  █████╗
        ████╗ ████║██╔══██╗████╗ ████║██╔══██╗██╔══██╗
        ██╔████╔██║███████║██╔████╔██║██████╔╝███████║
        ██║╚██╔╝██║██╔══██║██║╚██╔╝██║██╔══██╗██╔══██║
        ██║ ╚═╝ ██║██║  ██║██║ ╚═╝ ██║██████╔╝██║  ██║
        ╚═╝     ╚═╝╚═╝  ╚═╝╚═╝     ╚═╝╚═════╝ ╚═╝  ╚═╝

        mamba (0.7.12) supported by @QuantStack

        GitHub:  https://github.com/mamba-org/mamba
        Twitter: https://twitter.com/QuantStack

█████████████████████████████████████████████████████████████


Looking for: ['jupyter-lsp-python-lsp-server', 'jupyter-lsp-python-plugins']

conda-forge/noarch       [====================] (00m:01s) Done
conda-forge/linux-64     [====================] (00m:03s) Done
Encountered problems while solving.
Problem: package jupyter-lsp-python-lsp-server-1.2.0-pyhd8ed1ab_0 requires jupyter-lsp 1.2.0 pyhd8ed1ab_0, but none of the providers can be installed

```

**Comment by krassowski** [comment]
Yes, we knew that plugins would be a pain during the transition maybe we did it too fast before everyone caught up with it :( There is an issue open here: https://github.com/rupert/pyls-black/issues/35 and coc team also stumbled upon this here: https://github.com/yaegassy/coc-pylsp/issues/1

To avoid things like that it would be ideal we we could encourage the plugin authors to transition as well (or if they are no longer interested in maintenance, to move it to the community organization) or if they would prefer to stay with the unmaintained pyls version to forks the plugins. Not sure what is the best course of action now.

**Comment by bollwyvl** [comment]
Closing as resolved by #27
